### DISCUSSION TITLE
Reference for double inverted pendulum example

### DISCUSSION THREAD

**Discussion by ParamB11** [do-mpc/do-mpc#258] [Q&A]
The do-mpc website has an example of nlmpc for double inverted pendulum. In which K.E.-P.E. is used for the cost function and it is also mentioned that the standard quadratic cost function is ineffective. I was wondering if their are any references that support these two facts:
1. effectiveness of energy based cost function
2. ineffectiveness of standard quadratic cost function

**Comment by 4flixt** [comment]
Hi @ParamB11,

these should not be understood as claims but rather observations. We also took inspiration for this control design from this [paper](https://ieeexplore.ieee.org/abstract/document/973983?casa_token=t2LbDyt4dsQAAAAA:RcCPFpxL7hG-msKc3nHwRQyoRxowuHxilHf-gaPE0omXlv3EEceONCYrS5t1jCJmiYDR-CIymG4) which introduces the energy and passivity based objective. 

Best,
4flixt
